Java/app/build.gradle

```diff
@@ -41,7 +41,7 @@ dependencies {
     implementation 'androidx.annotation:annotation:1.1.0'
 
     //Taboola
-    implementation 'com.taboola:android-sdk-beta:meta-unified-1.0.0-570-8bc74b127'
+    implementation 'com.taboola:android-sdk-beta:meta-global-beta-1.0.4-610-b9f1650d1'
```

Comment:
@voltfmanSlava - Any reason why you are using the **global** version? Why not unified? This is a public repository and I am assuming we would want to not show a publisher specific version.

Reply:
@tomer-br-taboola updated to meta-unified-1.0.2-615-b6bbebdb0